cli.go

```diff
@@ -37,10 +38,15 @@ type Function interface {
 //
 // The Exec function never returns.
 func Exec(cmd Function) {
+	ExecContext(context.Background(), cmd)
```

Comment:
Should this be `context.TODO()` as well to be consistent with the `Call` function? Or did you have a reason in mind for using different background contexts in each case?